fetch/client/src/main/scala/endpoints4s/fetch/FetchUrl.scala

```diff
@@ -0,0 +1,3 @@
+package endpoints4s.fetch
+
+case class FetchUrl(val underlying: String) extends AnyVal
```

Comment:
What is the purpose of this class?

Reply:
Since `type Request[A] = A => (FetchUrl, FetchRequestInit)`, wanted to make it something stronger than a `String` or a type alias. What approach would you prefer here?

Reply:
OK I see. No problem but I would rather make this class `private[fetch]` then, if that works for you.

Reply:
Hmm, I don’t think it’s possible to make it private since it will part of the public signature of request constructors… It’s OK to leave it as it is. Just remove the `val` before the `underlying` field.